src/main/java/ch/wisv/areafiftylan/utils/mail/MailServiceImpl.java

```diff
@@ -173,7 +173,9 @@ public void sendTeamInviteMail(User user, String teamName, User teamCaptain) {
     @Override
     public void sendSeatOverrideMail(User user) {
         String subject = "Your seat was reset";
-        String message = "Your seat was reset by an admin.\nPlease log in to My Area to select your new seat.";
+        String message = "Unfortunately an admin had to reallocate your reserved seat.\n" +
```

Comment:
Just turn this into "we" instead of "an admin". Users don't really care about our internal hierarchy.